src/gridfs-stream/index.ts

```diff
@@ -44,8 +46,8 @@ export interface GridFSBucketPrivate {
     readPreference?: ReadPreference;
     writeConcern: WriteConcern | undefined;
   };
-  _chunksCollection: Collection;
-  _filesCollection: Collection;
+  _chunksCollection: Collection<GridFSChunk>;
```

Comment:
I decided to dogfood the generics changes in GridFS, it revealed a few type issues and is a good was to ensure we keep the generic usage type checked as we expect it to. I can revert if we want to do this separately.